hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/HapiClients.java

```diff
@@ -144,90 +152,252 @@ private void ensureChannelStubsInPool(@NonNull final NodeConnectInfo node, final
         stubSequences.putIfAbsent(channelUri, new AtomicInteger());
     }
 
-    private HapiClients(final List<NodeConnectInfo> nodes) {
-        this.nodes = nodes;
-        stubIds = nodes.stream().collect(Collectors.toMap(NodeConnectInfo::getAccount, NodeConnectInfo::uri));
-        tlsStubIds = nodes.stream().collect(Collectors.toMap(NodeConnectInfo::getAccount, NodeConnectInfo::tlsUri));
-        nodes.forEach(node -> {
+    private void ensureNodeOperatorChannelStubsInPool(@NonNull final HederaNode node) {
+        requireNonNull(node);
+        final var channelUri = uri(node.getHost(), node.getGrpcNodeOperatorPort());
+        final var existingPool = channelPools.computeIfAbsent(channelUri, COPY_ON_WRITE_LIST_SUPPLIER);
+        if (existingPool.size() < MAX_DESIRED_CHANNELS_PER_NODE) {
+            final var channel = createNettyChannel(false, node.getHost(), node.getGrpcNodeOperatorPort(), -1);
+            requireNonNull(channel, "Cannot continue without Netty channel");
+            existingPool.add(ChannelStubs.from(channel, new NodeConnectInfo(node.hapiSpecInfo()), false));
+        }
+        stubSequences.putIfAbsent(channelUri, new AtomicInteger());
+    }
+
+    private HapiClients(Supplier<List<NodeConnectInfo>> nodeInfosSupplier) {
```

Comment:
I wish we could make spotless do this... 🤔 

```suggestion
    private HapiClients(final Supplier<List<NodeConnectInfo>> nodeInfosSupplier) {
```
Not a big deal though

Reply:
Fixed :)